### PR TITLE
feat: Mongo DB backups proper implementation

### PR DESCRIPTION
**Pull request by marbonilla**
# Description
This pull request updates and corrects the following things:

1. The `Dockerfile` installs MongoDB version 4.4 instead of 6.0.
2. The `docker-entrypoint` corrects the construction of the `uri` string and renames variables in the azcopy command.
3. Some variable names and line breaks in the rendering of `multipurpose-jobs` are corrected.
4. Plugin variables and default values are updated, including the version of the [shipyard-utils](https://hub.docker.com/r/ednxops/shipyard-utils/tags) image.

## How to test it?
You can try it by activating the `drydock-backups` plugin, you can use [this steps and configuration](https://github.com/eduNEXT/drydock/tree/mar/az-backups/drydock_backups#drydock-backuyps).

### CODE REVIEW

File: drydock_backups/patches/drydock-multipurpose-jobs [hunk 1]
```diff
@@ -42,26 +44,27 @@ spec:
                   value: '{{ BACKUP_BUCKET_NAME }}'
                 - name: BUCKET_PATH
                   value: '{{ BACKUP_BUCKET_PATH }}'
+                - name: BACKUP_STORAGE_SERVICE
```
Comment (MAAngamarca) [inline drydock_backups/patches/drydock-multipurpose-jobs:47]: This variable is duplicated.

File: drydock_backups/plugin.py [hunk 2]
```diff
@@ -15,7 +15,7 @@
         "VERSION": __version__,
         "IMAGE": "ednxops/shipyard-utils:v1.1.0",
         "CRON_SCHEDULE": '0 2 * * *',
-        "BACKUP_STORAGE_SERVICE": "aws-s3",
+        "STORAGE_SERVICE": "aws-s3",
```
Comment (MAAngamarca) [inline drydock_backups/plugin.py:18]: is the name change correct?

Comment (marbonilla) [reply drydock_backups/plugin.py:18]: yes, if you prepend the string `BACKUP_` to all variables in the 'default' dictionary [here](https://github.com/eduNEXT/drydock/blob/mar/fix-drydock-backups/drydock_backups/plugin.py#L37-L43).

Comment (jfavellar90) [reply drydock_backups/plugin.py:18]: @marbonilla should you mark this comment as resolved?

**Review by jfavellar90 — APPROVED** [review]

**Review by MAAngamarca — APPROVED** [review]

**Pull request merged**